modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/notifications/NotificationsSettingsScreen.kt

```diff
@@ -94,6 +107,51 @@ private fun PreviewNotificationSettingsScreen(@PreviewParameter(ThemePreviewPara
     AppTheme(themeType) {
         NotificationsSettingsScreen(
             state = NotificationsSettingsViewModel.State(
+                areSystemNotificationsEnabled = true,
+                categories = listOf(
+                    CategoryModel(
+                        title = TextResource.fromText("My episodes"),
+                        preferences = listOf(
+                            NotificationPreferenceType.NotifyMeOnNewEpisodes(
+                                title = TextResource.fromText("Notify me"),
+                                isEnabled = false,
+                            ),
+                        ),
+                    ),
+                    CategoryModel(
+                        title = TextResource.fromText("Settings"),
+                        preferences = listOf(
+                            NotificationPreferenceType.PlayOverNotifications(
+                                title = TextResource.fromText("Play over notifications"),
+                                value = PlayOverNotificationSetting.DUCK,
+                                displayValue = TextResource.fromText("Duck"),
+                                options = emptyList(),
+                            ),
+                            NotificationPreferenceType.HidePlaybackNotificationOnPause(
+                                title = TextResource.fromText("Hide playback notification on pause"),
+                                isEnabled = true,
+                            ),
+                        ),
+                    ),
+                ),
+            ),
+            onPreferenceChanged = {},
+            onAdvancedSettingsClicked = {},
+            onBackPressed = {},
+            bottomInset = 0.dp,
+            onSelectRingtoneClicked = {},
+            onSelectPodcastsClicked = {},
+            onSystemNotificationsSettingsClicked = {},
+        )
+    }
+
+@Preview
+@Composable
+private fun PreviewDisabledNotificationSettingsScreen(@PreviewParameter(ThemePreviewParameterProvider::class) themeType: Theme.ThemeType) =
```

Comment:
To avoid the number of previews I typically use theme parameter for a screen with all it's content like banner shown, etc. And then just single-themed previews for all other scenarios. What do you think?